.github/workflows/kube-integration-tests-non-root.yaml

```diff
@@ -0,0 +1,104 @@
+name: Kube Integration Tests (Non-root)
+run-name: Kube Integration Tests (Non-root) - ${{ github.run_id }} - @${{ github.actor }}
+
+on:
+  push:
+    branches:
+      - master
+      - branch/*
+  pull_request:
+    paths:
+      - '.github/workflows/kube-integration-tests-non-root.yaml'
+      - '**.go'
+      - 'go.mod'
+      - 'go.sum'
+      - 'build.assets/Makefile'
+      - 'build.assets/Dockerfile*'
+      - 'Makefile'
+  merge_group:
+    paths:
+      - '.github/workflows/kube-integration-tests-non-root.yaml'
+      - '**.go'
+      - 'go.mod'
+      - 'go.sum'
+      - 'build.assets/Makefile'
+      - 'build.assets/Dockerfile*'
+      - 'Makefile'
+
+env:
+  TEST_KUBE: true
+  KUBECONFIG: /home/.kube/config
+
+jobs:
+  test:
+    name: Kube Integration Tests (Non-root)
+    if: ${{ !startsWith(github.head_ref, 'dependabot/') }}
```

Comment:
`if` conditionals do not need the `${{ }}` - they are always in expression context: https://docs.github.com/en/actions/learn-github-actions/expressions#about-expressions - I have no idea why that page says "you may omit the expression syntax  (`${{ }}` ) and the go on to give an example with that syntax :shrug:, but the fewer magical squiggles, the better I think.

Reply:
Oh! It's the leading bang! ! ! !

You could quote the string I guess, but probably makes a little more sense to leave the expression markers.